spinalcordtoolbox/process_seg.py

```diff
@@ -100,6 +100,20 @@ def compute_shape(segmentation, angle_correction=True, centerline_path=None, par
                                ncols=80):
         # Extract 2D patch
         current_patch = im_segr.data[:, :, iz]
+        # NOTE: `im_segr` is in the RPI orientation, so the first two dimensions are RL and AP.
```

Comment:
`im_segr` in the RPI orientation in FSLeyes:
![image](https://github.com/user-attachments/assets/0e437f96-80b8-4f64-a217-a1c82c8ddd02)